pkgs/applications/misc/cotp/default.nix

```diff
@@ -7,7 +7,7 @@
 }:
 
 rustPlatform.buildRustPackage rec {
-  name = "cotp";
```

Comment:
the cargo hashes of rust packages needs to be updated since it references the name of the derivation

Reply:
Aren't the hash only depending on the content of the resulting vendor directory?

Reply:
it might be caused by this: https://github.com/NixOS/nixpkgs/blob/9b97ad7b4330aacda9b2343396eb3df8a853b4fc/pkgs/build-support/rust/fetch-cargo-tarball/default.nix#L94

Reply:
Fixed. Thank you!